pupil/public/javascripts/mtc-check.js

```diff
@@ -1,25 +1,24 @@
-/* global Touch DocumentTouch Event */
-
 (function (global) {
   'use strict'
-  var GOVUK = global.GOVUK || {}
-  var startTime
-  var pageInterval = 5  // seconds allowed per page
-  var preloadInterval = 2 // seconds for student preparation
-  var deadline
 
-  var registerInputs = []
+  var GOVUK = global.GOVUK || {}
+  var defaultQuestionTimeLimit = 5  // seconds allowed per page
```

Comment:
Server side, defaults are stored in config, front-end wise this default value should never be needed but it's in place for peace of mind.